packages/elements/src/slider/__test__/slider.event.test.js

```diff
@@ -0,0 +1,1465 @@
+import {
+  fixture,
+  expect,
+  elementUpdated,
+  oneEvent,
+} from '@refinitiv-ui/test-helpers';
+
+import '@refinitiv-ui/elements/slider';
+import '@refinitiv-ui/elemental-theme/light/ef-slider';
+
+import { tabSliderPosition, calculateValue } from './utils';
+
+describe('slider/Events', () => {
+  describe('Events', () => {
```

Comment:
I did not find any other `describe` in this file. You can either split other test and give describe a reasonable name or remove it.

Reply:
Done.